SlugifierInterface.php

```diff
@@ -9,21 +9,31 @@
  * file that was distributed with this source code.
  */
 
+namespace Symfony\Cmf\Bundle\CoreBundle\Slugifier;
+
+if (!interface_exists('Symfony\Cmf\Bundle\CoreBundle\Slugifier\SlugifierInterface')) {
+    /**
+     * @deprecated Since 1.0, to be removed in 2.0.
+     */
+    interface SlugifierInterface {
```

Comment:
not sure which is better (or rather, less bad). @lsmith77 do you have an opinion?